internal/store/driver.go

```diff
@@ -844,11 +870,17 @@ func (d *Driver) calculateHTTPSEdgesFromIngress(edgeMap map[string]ingressv1alph
 					continue
 				}
 
-				policyStr, err := json.Marshal(modSet.Modules.Policy)
+				var policyStr []byte
 
-				if err != nil {
-					d.log.Error(err, "cannot convert module-set policy json", "Policy", modSet.Modules.Policy)
-					continue
+				// Handle deciding which policy we want to serialize
+				if trafficPolicy == nil {
```

Comment:
this always felt a little weird, to check trafficpolicy to see if we use modset. it may work ok, but might flow better if we check if trafficPolicy not nil, if so then use trafficPolicy. fallback to check if modules.policy is not nil, then use modules.policy. so everything is nil checked right before use.

Reply:
Made this change, ran a make deploy and didn't see my policy on the edge. Same whne I stashed the changes. Working on deleting the `minikube` and starting fresh.

Reply:
@bobzilladev hot off the press: https://github.com/ngrok/kubernetes-ingress-controller/pull/370/commits/de91cbcc6c8da8f111a3f9c5c20b793e90e34d1b